test/JDBC/expected/BABEL-3655.out

```diff
@@ -0,0 +1,46 @@
+-- tsql
```

Comment:
Let's write an MVU test instead of a normal test (Ref: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/764/files)

Reply:
Is there any particular reason that we want to have MVU test here? We are not altering any pre-defined BBF objects and thus don't need to test dependency. 
For functionality test I think normal JDBC would be enough?

Reply:
(correction)
Usually, an MVU test covers both normal and MVU scenario, but in here, we need both a normal and an MVU test because of single/multi-db settings. More precisely,
- normal test: need to set `migration_mode` into `multi-db` at the beginning and reset it to `single-db` at the end
- MVU test: our MVU test framework is using `multi-db` migration mode, so we don't need to change the `migration_mode`

In other words, we should have `BABEL-3655.mix` and `BABEL-3655-vu-{prepare/verify/cleanup}.sql`

Reply:
Yes I agree, I've asked Ray to do the same.

Reply:
FYI, adding an MVU test is almost always beneficial in terms of testing coverage. A broader testing coverage can help future development from various regressions as well. Actually, we have a plan to migrate all normal tests into MVU tests (not sure when we do it though).